src/main/java/io/jenkins/infra/repository_permissions_updater/github_team_sync/YmlTeamLoader.java

```diff
@@ -0,0 +1,64 @@
+package io.jenkins.infra.repository_permissions_updater.github_team_sync;
+import org.yaml.snakeyaml.Yaml;
+import java.io.FileInputStream;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class YmlTeamLoader {
```

Comment:
yaml would be clearer / easier to read
```suggestion
public class YamlTeamLoader {
```